mod/groups/views/default/groups/js.php

```diff
@@ -1,9 +1,11 @@
 <?php
 /**
- * Javascript for Groups forms
- *
  * @package ElggGroups
+ * @deprecated 2.2
  */
+elgg_deprecated_notice('groups/js view has been deprecated. Instead define dependency on "groups/navigation" AMD module for "feature" and "unfeature" menu items', '2.2');
```

Comment:
Instead of this notice, it'd be more reliable to put this in the init:

```php
if (_elgg_view_may_be_altered('groups/js', __DIR__ . '/views/default/groups/js.php')) {
    // show deprecation notice.
}
```

(We'd also teach `_elgg_view_may_be_altered` to accept an absolute path as 2nd arg)